File: html5ajax/server/classes/employee.php
<?php

    class Employee {
        public $name;
        public $address;
        public $salary;
        
        public function getName() {
            return $name;
        }
        
        public function getAddress() {
            return $address;
        }
        
        public function getSalary() {
            return $salary;
        }
        
        public function setName($name) {
            $this->name = $name;
        }
        
        public function setAddress($address) {
            $this->address = $address;
        }
        
        public function setSalary($salary) {
            $this->salary = $salary;
        }
    }
?>

File: contributions/egl/src/Employee.java
public   class  Employee {
	
  
  public float getSalary  () {
    return salary;
  }

	

  public void setSalary  (float salary) {
    this.salary = salary;
  }

	
	
	public void cutSalary() {
		setSalary (getSalary() / 2);
	}

	
  private java.lang.String name;

	
  
  public java.lang.String getName () {
    return name;
  }

	

  public void setName (java.lang.String name) {
    this.name = name;
  }

	  
  
  private java.lang.String address;

	
  
  public java.lang.String getAddress () {
    return address;
  }

	

  public void setAddress (java.lang.String address) {
    this.address = address;
  }

	  
  
  private float salary;


}
